src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -26,6 +26,12 @@ private static string GetFileNameFromResourceName(string s)
 
         public int CreateEmptyProject(string languageName, string templateDir)
         {
+            if (File.Exists(Path.Combine(Directory.GetCurrentDirectory(), "project.json")))
+            {
+                Reporter.Error.WriteLine($"Creating new {languageName} project failed, project.json already exists.");
```

Comment:
@blackdwarf Should we voice this out like this? Or should we say folder already contains a project in it? Also, is this enough check or should we check if the current folder is empty (I particularly don't like this check in VS, btw).